Sources/Player/PlaybackEngine/Internal/PlayerItemLoader.swift

```diff
@@ -42,7 +49,8 @@ private extension PlayerItemLoader {
 
 		if PlayerWorld.developmentFeatureFlagProvider.isOffliningEnabled,
 		   let offlineEntry = try? offlineStorage?.get(key: mediaProduct.productId),
-		   let offlinedMediaProduct = PlayableOfflinedMediaProduct(from: offlineEntry)
+		   let offlinedMediaProduct = PlayableOfflinedMediaProduct(from: offlineEntry),
+		   offlinePlaybackPrivilegeCheck?() ?? true == true
```

Comment:
[suggestion] tiny suggestion to maybe store the result `offlinePlaybackPrivilegeCheck?() ?? true` to later compare in the if condition, i think it could improve slightly the readability of the condition.

Reply:
yeah agree. 

I made this PR ready so I could check the linter issue on an old PR, but I want to make some more changes to it still

Reply:
ahh, sorry about it, just let me know when ready to be checked